src/logger.ts

```diff
@@ -0,0 +1,71 @@
+import { Configuration } from './Configuration';
+import { Level } from './level';
+import { getColor } from './level';
+import { getName } from './level';
+const fs = require('fs');
+
+export class Logger{
```

Comment:
It would be better if you devide the logger to ConsoleLogger and FileLogger that inherire from base Logger class(That implement Logger interface), instead of devide it inside the class with if else statement.